### DISCUSSION TITLE
UTM didn't have full screen access to the external monitor

### DISCUSSION THREAD

**Discussion by Andy312432** [utmapp/UTM#4411] [UTM Help (iOS)]
![PXL_20220912_152034402](https://user-images.githubusercontent.com/31586206/189693835-4f04119a-c0e9-4902-b124-fd8ffc657cb7.jpg)

Like these two images. When I share the screen via YouTube or bilibili, it can have full screen access via airplay. Is there a tweak or sth to bypass ios15.1 limitations?

![PXL_20220912_152012862](https://user-images.githubusercontent.com/31586206/189693882-826690f6-2852-46be-befc-0d9e2c1b0eec.jpg)

**Configuration (required)**
* UTM Version: 4.0.4
* OS Version: 15.1
* Device Model: iPad pro m1 3rd gen 
* Is it jailbroken (name jailbreak used)? No
* How did you install UTM? Trollstore

**Comment by osy** [comment]
I'm not sure what the issue is here.

**Reply by Andy312432** [reply]
I’m not familiar with ios programming, but maybe is that currently UTM didn’t read the screen resolution (canva) size before switching to it? 
Not sure though…

**Reply by Andy312432** [reply]
I mean, UTM didn’t reclaim the display canva after connected it to external display…?

**Reply by osy** [reply]
I’m not sure what your screenshot is supposed to show though.

**Reply by osy** [reply]
Oh! I see the black bars now.

**Reply by ak4zh** [reply]
Yes the issue is black bars.
Also the external display just mirrors, is there a way to use it as extended display?
Apps like Youtube and Luma Fusions can work in full screen but other apps does not so may be something can be done with trollstore, these apps probably have some additional entitlements by apple.

**Comment by Andy312432** [comment]
Guess we should reopen this issue...?
@osy (sorry for ping)

**Reply by osy** [reply]
Oh I forgot about this because you didn’t post it as an issue but as a discussion. I’ll convert it now.

**Reply by FacetiouslyWry** [reply]
nice
